kimchi/src/circuits/polynomials/permutation.rs

```diff
@@ -63,9 +64,6 @@ use rand::{CryptoRng, RngCore};
 use rayon::prelude::*;
 use std::array;
 
-/// Number of constraints produced by the argument.
-pub const CONSTRAINTS: u32 = 3;
```

Comment:
same here, this shouldn't move

Reply:
Done https://github.com/o1-labs/proof-systems/pull/974/commits/91350343175bf874d16ada51a0ee486a6135db73